ringhold/README.md

```diff
@@ -1,6 +1,6 @@
-# Storage Inventory local artifact removal process (ringhold)
+# Storage Inventory local artifact deletion process (ringhold)
 
-Process to remove local artifacts that are no longer being synchronised by fenwick. This tool is used
+Process to delete local artifacts that are no longer being synchronised by fenwick. This tool is used
```

Comment:
Wasn't part of this work specifically, but this change from remove to delete makes it more obvious that this is a little too vague. Maybe something like:
```
Process to remove the local copy of artifacts from a storage site inventory database and generate DeletedStorageLocationEvent(s) so the removal will propagate correctly to a global inventory. This does not remove the files from storage (see `tantar`).
```
I mentioned "storage site" explicitly because this tool doesn't really do the right thing in global (there's a future story for that).

Reply:
Thought we said it should be delete instead of remove, to be more explicit what it is doing., but the above is much better. Updated.